src/duct.py

```diff
@@ -268,6 +268,7 @@ def create_and_parse_args():
         choices=["all", "system-summary", "processes-samples"],
         help="Record system-summary, processes-samples, or all",
     )
+    parser.add_argument("inner_args", nargs=argparse.REMAINDER)
```

Comment:
Shouldn't the `nargs` be applied to the `arguments` argument above instead of adding this?

Reply:
Nice catch.